### PR TITLE
Added Python Basics folder

### PR DESCRIPTION
**Pull request by keivalya**
Made certain changes:
- [x] Added folder
- [x] Added comments to code blocks
- [x] Improved readability

### CODE REVIEW

File: Pandas Basics/Pandas_basics.ipynb [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+      "metadata": {
+        "id": "cG-b90nWkdaK",
+        "outputId": "de28c3e4-2cff-4851-caaf-1f51dd7c03a3",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 118
+        }
+      },
+      "source": [
+        "# Q: Create a table listing: Country, Capital, Area, Population\n",
```
Comment (achoudh5) [inline Pandas Basics/Pandas_basics.ipynb:47]: Looks better, can you also copy and paste this line like in between all code maybe like for this one, between line 45 and 46? Similarly for **others too.**

**Review by achoudh5 — CHANGES_REQUESTED** [review]
Suggested a few changes in terms of where to put the questions! We want maximum user visibility without complexity :)

File: Pandas Basics/Pandas_basics.ipynb (hunk 1 above)
Comment (keivalya) [reply Pandas Basics/Pandas_basics.ipynb:47]: Well, I checked doing as asked me to.
But apparently, I'm using Colab Notebook and changing that might disrupt the structure of code.
Yet I found a small issue in it. I'll change that.

If you think there is still scope to make it better, do let me know.

And yes, spell check on the label: hacktoberfest-accepted

**Comment by achoudh5** [comment]
Thank you for your contribution @keivalya !

**Pull request merged**